submarine-experiment-prehandler/prehandler_main.py

```diff
@@ -0,0 +1,22 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+from fs_prehandler.hdfs_prehandler import HDFSPreHandler 
+
+if __name__ == "__main__":
+    fsPrehandler = HDFSPreHandler()
```

Comment:
We could add a new function `get_prehandler` to create corresponding filesystems. Create an HDFS handler if the uri prefix is `hdfs://`or an s3 handler if the uri prefix is`s3://`. What do you think?
```suggestion
    fsPrehandler = get_prehandler(uri=uri)
```

Reply:
I think its a good approach if we leverage some kind of factory pattern here, but I thought maybe we could use another environment to determine the instance we get here? Because user may need setup different configurations for different file systems, and our UI also need to provide specific  fields for them.